### PR TITLE
Log asset name for RAFT debug/warn/error messages

### PR DESCRIPTION
**Pull request by neilalexander**
This should make it considerably easier to identify which clustered streams/consumers a given RAFT log line applies to without having to dissect the file store.

Signed-off-by: Neil Twigg <neil@nats.io>

/cc @nats-io/core

### CODE REVIEW

File: server/jetstream_cluster.go [hunk 1]
```diff
@@ -1696,7 +1696,7 @@ func (js *jetStream) createRaftGroup(accName string, rg *raftGroup, storage Stor
 	}
 	js.mu.Unlock()
 
-	s.Debugf("JetStream cluster creating raft group:%+v", rg)
+	s.Debugf("JetStream cluster creating raft group:%+v for asset %q", rg, assetName)
```
Comment (ripienaar) [inline server/jetstream_cluster.go:1699]: would be sweet if in the `StreamConfig` that is used to make the `S-` and `C-` streams we use the new metadata to also record the asset and account the raft asset relates to.

Comment (derekcollison) [reply server/jetstream_cluster.go:1699]: 2.10?

File: server/jetstream_cluster.go [hunk 2]
```diff
@@ -1672,7 +1672,7 @@ func (rg *raftGroup) setPreferred() {
 }
 
 // createRaftGroup is called to spin up this raft group if needed.
-func (js *jetStream) createRaftGroup(accName string, rg *raftGroup, storage StorageType) error {
+func (js *jetStream) createRaftGroup(accName, assetName string, rg *raftGroup, storage StorageType) error {
```
Comment (derekcollison) [inline server/jetstream_cluster.go:1675]: Why not just use cfg.Name?

**Review by derekcollison — CHANGES_REQUESTED** [review]
We should just have the debug line figure out on the fly which name to use instead of passing things around IMO.

File: server/jetstream_cluster.go (hunk 1 above)
Comment (ripienaar) [reply server/jetstream_cluster.go:1699]: right didnt see this targetted main, fair.

**Comment by derekcollison** [comment]
Any updates here?

File: server/jetstream_cluster.go (hunk 2 above)
Comment (neilalexander) [reply server/jetstream_cluster.go:1787]: `cfg.Name` is set from `rg.Name`, which seems to be the raft ID, not the actual asset name.

Comment (derekcollison) [reply server/jetstream_cluster.go:1787]: Yes its the store name. But if you are trying to tie these together at this level we would need to be more expressive then assetName I believe.

File: server/jetstream_cluster.go (hunk 1 above)
Comment (derekcollison) [reply server/jetstream_cluster.go:1811]: Yes this will need to be for dev.

File: server/jetstream_cluster.go (hunk 2 above)
Comment (derekcollison) [reply server/jetstream_cluster.go:1787]: E.g. a consumer would also need which stream it belongs to.

Let's chat about this a bit on a Zoom at some point.

**Pull request closed without merging**